core/assets/bundles/bundle_de.properties

```diff
@@ -793,7 +795,7 @@ bullet.damage = [stat]{0}[lightgray] Schaden
 bullet.splashdamage = [stat]{0}[lightgray] Flächenschaden ~[stat] {1}[lightgray] Kacheln
 bullet.incendiary = [stat]entzündend
 bullet.homing = [stat]zielsuchend
-bullet.frag = [stat]explosiv
+bullet.frag = [stat]{0}[lightgray]x explosive Projektile:
```

Comment:
lowercase explosive & uppercase projectile?

Reply:
that's german for you